source/nodejs/tests/unit-tests/template-test-resources/template-array-complex.json

```diff
@@ -0,0 +1,220 @@
+[
+    {
+        "type": "AdaptiveCard",
+        "$when": "${!$host.carousel}",
+        "body": [
+            {
+                "type": "TextBlock",
+                "text": "This is the regular card",
+                "wrap": true
+            },
+            {
+                "type": "TextBlock",
```

Comment:
if we're really going for complex here, consider using the templated version of this sample and adding the `"carousel": true|false;` to the data blob...